loadtest/user/userentity/actions.go

```diff
@@ -696,6 +696,38 @@ func (ue *UserEntity) GetChannelMembers(channelId string, page, perPage int) err
 	return ue.store.SetChannelMembers(channelMembers)
 }
 
+// GetAllChannelMembersForUser gets all channel memberships for the
+// specified user regardless of the team the channels are part of
+func (ue *UserEntity) GetAllChannelMembersForUser(userId string) error {
+	perPage := 200
+	page := 0
```

Comment:
Apologies for some more work. But I actually improved the API by streaming all data if you set page=-1: https://github.com/mattermost/mattermost/pull/30771/files#diff-f49fddb8b3d75c63925d36a97a8d5389337826b11ef544950f4898202d142489R6677.

So we have 2 options:
1. Merge this PR now and revert the changes that you made.
2. Or wait till my PR is merged, and update the API to use -1 as page number.

I am fine with either. Up to you.

Reply:
Oh, nice! No need to be sorry, that's actually a great improvement, thanks! I'll go with 2., will wait until your PR is merged and I'll make the change in the meantime.

Reply:
Actually, we'll need both implementations to support older server versions.

Reply:
Check https://github.com/mattermost/mattermost-load-test-ng/pull/908/commits/ffc0f7fa1c6a303161bdc7869bdfa7fdfafaf9a7 As the client itself has not changed, there is no need to wait for your PR to be merged, assuming it will be released in 10.9.0.